Flaky test TestCassandraTokenSplitManager

```
    Tests run: 749, Failures: 1, Errors: 0, Skipped: 7, Time elapsed: 465.517 sec <<< FAILURE! - in TestSuite
    testNonEmptyTable(com.facebook.presto.cassandra.TestCassandraTokenSplitManager)  Time elapsed: 0.309 sec  <<< FAILURE!
    java.lang.AssertionError: expected [10] but found [1]
        at com.facebook.presto.cassandra.TestCassandraTokenSplitManager.testNonEmptyTable(TestCassandraTokenSplitManager.java:69)
```

--- a/presto-cassandra/src/test/java/com/facebook/presto/cassandra/EmbeddedCassandra.java
+++ b/presto-cassandra/src/test/java/com/facebook/presto/cassandra/EmbeddedCassandra.java
@@ -77,14 +77,14 @@ public final class EmbeddedCassandra
         CassandraDaemon cassandraDaemon = new CassandraDaemon();
         cassandraDaemon.activate();
 
-        Cluster cluster = Cluster.builder()
+        Cluster.Builder clusterBuilder = Cluster.builder()
                 .withProtocolVersion(V3)
                 .withClusterName("TestCluster")
                 .addContactPointsWithPorts(ImmutableList.of(
                         new InetSocketAddress(HOST, PORT)))
-                .withMaxSchemaAgreementWaitSeconds(30)
-                .build();
+                .withMaxSchemaAgreementWaitSeconds(30);
 
+        ReopeningCluster cluster = new ReopeningCluster(clusterBuilder::build);
         CassandraSession session = new NativeCassandraSession(
                 "EmbeddedCassandra",
                 JsonCodec.listJsonCodec(ExtraColumnMetadata.class),

--- a/presto-cassandra/src/test/java/com/facebook/presto/cassandra/EmbeddedCassandra.java
+++ b/presto-cassandra/src/test/java/com/facebook/presto/cassandra/EmbeddedCassandra.java
@@ -44,6 +44,7 @@ import static java.nio.charset.StandardCharsets.UTF_8;
 import static java.util.Objects.requireNonNull;
 import static java.util.concurrent.TimeUnit.MINUTES;
 import static java.util.concurrent.TimeUnit.NANOSECONDS;
+import static java.util.concurrent.TimeUnit.SECONDS;
 import static org.testng.Assert.assertEquals;
 
 public final class EmbeddedCassandra
@@ -167,6 +168,7 @@ public final class EmbeddedCassandra
                 return;
             }
             log.info("Size estimates haven't been refreshed as expected. Retrying ...");
+            SECONDS.sleep(1);
         }
         throw new TimeoutException(format("Attempting to refresh size estimates for table %s.%s has timed out after %s", keyspace, table, REFRESH_SIZE_ESTIMATES_TIMEOUT));
     }

--- a/presto-cassandra/src/test/resources/cu-cassandra.yaml
+++ b/presto-cassandra/src/test/resources/cu-cassandra.yaml
@@ -19,7 +19,7 @@ cluster_name: 'Test Cluster'
 # the heaviest-loaded existing node.  If there is no load information
 # available, such as is the case with a new cluster, it will pick
 # a random token, which will lead to hot spots.
-#initial_token:
+initial_token: -9223372036854775808
 
 # See http://wiki.apache.org/cassandra/HintedHandoff
 hinted_handoff_enabled: false

